app/services/mixtape_generator.rb

```diff
@@ -0,0 +1,20 @@
+class MixtapeGenerator
+  def initialize(genre:)
+    @genre = genre
+  end
+
+  def self.perform(genre:)
+    new(genre: genre).perform
+  end
+
+  def perform
+    Mixtape.new(genre: @genre, songs: pick_songs)
+  end
+
+  def pick_songs
```

Comment:
Need to try out bullet-- I'll look again but I think I remember leaving out ```includes``` and seeing 50 queries scroll by in my rails server window.